frontend/scss/components/molecules/format-filter-hint.scss

```diff
@@ -13,10 +13,14 @@ $height: 60px;
   height: $height;
   color: color('white');
   background-color: color('ebony-clay');
+  position: sticky!important;
```

Comment:
this is unfortunately not valid AMP

Reply:
Using important?

Reply:
https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages/